orchestrator/api/helpers.py

```diff
@@ -52,6 +52,26 @@ def _process_text_query(q: str) -> str:
         return q
 
 
+def _add_sort_to_query(query: Query, sort: Optional[List[str]]) -> Query:
+    if sort is None or len(sort) < 2:
+        return query
+
+    for item in chunked(sort, 2):
+        if item and len(item) == 2:
+            order_by_expr = expression.desc if item[1].upper() == "DESC" else expression.asc
+            if item[0] in ["product", "tag"]:
+                field = "name" if item[0] == "product" else "tag"
+                col = ProductTable.__dict__.get(field)
+            else:
+                col = SubscriptionTable.__dict__.get(item[0])
+
+            if col:
+                query = query.order_by(order_by_expr(col))
+            else:
+                logger.warn(f"Unable to add sort. Unknown field: {item[0]}")
```

Comment:
I don't think we have any use in logging this warning, it's the user (or at least client) who made an error. And imo we should return a 400 error instead of 200

Reply:
Done